Resources/config/oro/routing.yml

```diff
@@ -1,4 +1,4 @@
 akeneo_integration_shipping:
-  resource:     "@OroAkeneoBundle/Controller/ValidateConnectionController.php"
-  type:         annotation
-  prefix:       /akeneo-validate
```

Comment:
please add empty line at the end of file

Reply:
Done